src/lightly_train/_commands/export_task.py

```diff
@@ -63,6 +75,7 @@ def export_onnx(
     num_channels: int = 3,
     height: int = 224,
     width: int = 224,
+    precision: OnnxPrecision = OnnxPrecision.F32_TRUE,
```

Comment:
We should not force the user to pass enums in public functions as that makes using the function more complicated and users have to learn about the enum class and where to import it from. Instead we use literals. See for example `train.py` or `train_task.py`.

Without enums:
```
export_onnx(..., precision="16-true")
```

With enums:
```
from lightly_train._commands.export_task import OnnxPrecision

export_onnx(..., precision=OnnxPrecision.F32_TRUE)
```

Internally we can use the enum. Pydantic should be able to convert the string value automatically to it.

Reply:
Done